subsys/net/l2/ieee802154/ieee802154_shell.c

```diff
@@ -214,9 +214,10 @@ static void scan_result_cb(struct net_mgmt_event_callback *cb,
 	char buf[64];
 
 	shell_fprintf(cb_shell, SHELL_NORMAL,
-		      "\nChannel: %u\tPAN ID: %u\tCoordinator Address: %s\t "
-		      "LQI: %u\n", params.channel, params.pan_id,
-		      print_coordinator_address(buf, sizeof(buf)), params.lqi);
+		      "Channel: %u\tPAN ID: %u\tCoordinator Address: %s\t "
+		      "LQI: %u Associable: %s\n", params.channel, params.pan_id,
```

Comment:
I personally would prefer "association permitted" for improved consistency and standard compliance (strictly optional).

Reply:
I disliked it, because it is a long name and I don't want to clutter the shell to much, as it is already a pretty long prompt.